src/sig_stfl/lms/external/hss_verify_inc.h

```diff
@@ -20,7 +21,8 @@
  *    hss_validate_update( &ctx, message_part_2, len_2 );
  *    hss_validate_update( &ctx, message_part_3, len_3 );
  *    success = hss_validate_finalize( &ctx, signature );
- *    if (success) printf( "The signature validated\n" );
+ *    if (success) printf( "The signature validated
```

Comment:
Did something go wrong with the `\n` symbol here?

Reply:
Yep, will fix.